src/components/Icon/OutwardArrowsIcon/OutwardArrowsIcon.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Icon from '../Icon';
+import { lucidClassNames } from '../../../util/style-helpers';
+import { createClass } from '../../../util/component-types';
+
+const { oneOf } = React.PropTypes;
+
+const cx = lucidClassNames.bind('&-OutwardArrowsIcon');
+
+const paths = {
```

Comment:
I don't think so, it could be `diagonal-inverted` 🚎 

But yeah, I don't think so.